package.json

```diff
@@ -130,6 +130,7 @@
     "react-native-vector-icons": "^10.2.0",
     "react-native-vision-camera": "4.0.5",
     "react-native-volume-manager": "^1.10.0",
+    "react-native-walkthrough-tooltip": "^1.6.0",
```

Comment:
I'm increasingly anxious about the vast number of libraries we are struggling to maintain. What's the main reason to use this instead of using the modal library we already have? Positioning? Did you try an implementation with modals?